Author: Bernhard M. Wiedemann <bwiedemann suse.de>
Date: 2018-08-17

https://bugzilla.opensuse.org/show_bug.cgi?id=1100677 packages do not build reproducibly from compile-time CPU-detection


Always build with SSE2, independent of build system CPU

Index: SConstruct
===================================================================
--- SConstruct.orig	2019-01-03 16:18:09.167995690 +0200
+++ SConstruct	2019-01-03 16:18:10.556060875 +0200
@@ -748,16 +748,10 @@ def cc_flags_x86(cpuinfo, enable_optimiz
     opt_flags = [
             ('sse', ('-mfpmath=sse', '-msse')),
             ('sse2', '-msse2'),
-            ('ssse3', '-mssse3'),
-            ('sse4', '-msse4'),
-            ('sse4_1', '-msse4.1'),
-            ('sse4_2', '-msse4.2'),
-            ('sse4a', '-msse4a'),
-            ('3dnow', '-m3dnow'),
     ]
     if enable_optimizations:
         for flag, gccopt in opt_flags:
-            if flag in cpuinfo.x86_flags:
+            if True:
                 if isinstance(gccopt, (tuple, list)):
                     cc_opts.extend(gccopt)
                 else:
